### PR TITLE
feature: support shm size

### PR DESCRIPTION
**Pull request by Ace-Tang**
Signed-off-by: Ace-Tang <aceapril@126.com>

### Ⅰ. Describe what this PR did

support run commond `pouch run -d --shm-size 1g  registry.hub.docker.com/library/busybox:latest top`

### CODE REVIEW

File: daemon/mgr/spec_mount.go [hunk 1]
```diff
@@ -86,6 +91,10 @@ func setupMounts(ctx context.Context, c *Container, s *specs.Spec) error {
 
 		// TODO: support copy data.
 
+		if mp.Destination == "/dev/shm" && c.HostConfig.ShmSize != nil {
```
Comment (fuweid) [inline daemon/mgr/spec_mount.go]: Hi @Ace-Tang , I checked the swagger and found `If omitted, the system uses 64MB`. Is it correct for the API side? or it's just for the CLI?

File: cli/container.go [hunk 2]
```diff
@@ -172,6 +173,11 @@ func (c *container) config() (*types.ContainerCreateConfig, error) {
 		return nil, err
 	}
 
+	shmSize, err := opts.ParseShmSize(c.shmSize)
```
Comment (fuweid) [inline cli/container.go]: we are missing the `opts.ParseShmSize` function 😄

Comment (Ace-Tang) [reply cli/container.go]: Oh, yes

File: daemon/mgr/spec_mount.go (hunk 1 above)
Comment (Ace-Tang) [reply daemon/mgr/spec_mount.go]: I think the swagger readme is copy from something where, and 64m is not implement in code

**Comment by Ace-Tang** [comment]
Oh, CI fails because of this case for many times.
```
FAIL: /go/src/github.com/alibaba/pouch/test/cli_restart_test.go:34: PouchRestartSuite.TestPouchRestart
```

File: daemon/mgr/spec_mount.go (hunk 1 above)
Comment (fuweid) [reply daemon/mgr/spec_mount.go]: OK. So could you mind to help us to update the description? 😄

Comment (fuweid) [reply daemon/mgr/spec_mount.go]: yeah....containerd sets it to 64M. 

https://github.com/containerd/containerd/blob/master/oci/spec_unix.go#L129-L133

So, we keep it on the swagger.

**Comment by fuweid** [comment]
LGTM

**Pull request merged**